python/metatensor-torch/metatensor/torch/atomistic/units.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+import torch
+
+
+@torch.jit.script
+class Quantity:
+    baseline: str
+    """TODO"""
+
+    conversions: Dict[str, float]
+    """TODO"""
+
+    def __init__(self, baseline: str, conversions: Dict[str, float]):
+        self.baseline = baseline
+        self.conversions = conversions
+
+        assert self.baseline in self.conversions
+        assert self.conversions[self.baseline] == 1.0
+
+
+KNOWN_QUANTITIES = {
+    "length": Quantity(
+        baseline="angstrom",
+        conversions={
+            "angstrom": 1.0,
+            "bohr": 0.529177249,
+            "nanometer": 10.0,
+            "nm": 10.0,
+        },
+    ),
+    "energy": Quantity(
+        baseline="ev",
+        conversions={
+            "ev": 1.0,
+            "mev": 1e-3,
+            "hartree": 27.211396641308,
```

Comment:
kj/mol maybe

Reply:
yes, we'll need a lot more units & quantities here!